pages/views/roles/data.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: default
+title: Data View
+permalink: views/roles/data
+---
+
+# Data View
+
+{% include carbonStandard.html variant=site.data.overlays.data hideOutOfScope=true %}
+
+Every bit of data, at rest or in transit, incurs a carbon cost.
+If a user minimises the amount of data they store, then, they can significantly reduce carbon emissions due to storage. Doing so is also likely to reduce monetary costs: hosted services generally charge by data use, while local storage will use less energy to read and write the data, and can use fewer or smaller storage options, reducing embedded carbon.
+
+## Upstream Emissions
+
+## Operational Emissions
+
+While upstream data has a carbon cost, it is operational emissions where a company can make the choices for the greatest effect.
+
+The first, and perhaps biggest choice, is where to store data: on local servers, or on the cloud (or some other hosted solution).
+
+In either case, though, costs scale with the amount of data stored (and, to a lesser extent, with the rate at which that data is written or read). Therefore, compressing the data is likely to reduce storage costs; however, it will incur extra emissions in computing for compression and decompression.
+
+### Direct
+{% include categoryLabel.html label="CatO" %}
+{% include categoryItem.html item="CatONetworkDevices" id="NetworkDevices" %}
+
+Transferring data over networks incurs an electricity cost, and therefore a carbon cost. 
+
+{% include categoryItem.html item="CatOServers" id="Servers" %}
+
+When hosting locally, the biggest factor in emissions is likely to be the embodied carbon of the servers.
+
+| Storage Type            | Embedded Carbon (kgCO2e/TB) | Operating Power (W/TB) |
+| ----------------------- | --------------------------: | ---------------------: |
+| Hard Disk Drive (HDD)   | 38                          | 1                      |
+| Solid State Drive (SSD) | 316                         | 2.5                    |
+
+(These figures are for a 8 TB HDD and a 1 TB SSD. Of course, there are reasons for using SSDs over HDDs, such as faster read/write speed.)
+
+A HDD will therefore have roughly equal amortised embodied carbon and operating emissions over 10 years of constant operation (while a SSD would require 35 years!). Since most storage media have a typical use life of around 5 years, increasing the lifespan of storage media is likely to have a significant effect on average emissions. 
+
+#### Long-term Copies and Backups
```

Comment:
H4 don't appear to render very clearly as a heading - something we'll need to look at in another ticket.